create-publication-from-scopus/src/main/java/no/sikt/nva/scopus/conversion/ContributorExtractor.java

```diff
@@ -160,6 +163,7 @@ private Identity generateContributorIdentityFromAuthorTp(AuthorTp authorTp) {
         var identity = new Identity();
         identity.setName(determineContributorName(authorTp));
         identity.setOrcId(getOrcidAsUriString(authorTp));
+        identity.setId(piaConnection.getCristinID(authorTp.getAuid()));
```

Comment:
For meg ser det ut som PiaConnection kaster exception ved feil i kommunikasjonen som ikke håndteres her. Det vil få store konsekvenser ettersom hele publikasjonen vil forkastes. Kanskje vi heller bør håndtere det og ikke sette identity for authors i dette tilfellet?

Mulig jeg ikke forstår hele bildet og at dette er greit?

Reply:
Jeg tror også det er lurere å heller la være å sette identity isteden for å forkaste publikasjonen i det tilfellet.

Reply:
Fikset